package.json

```diff
@@ -9,9 +9,12 @@
     "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-android'",
     "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
     "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-ios'",
+    "build:android:release": "cd android && ./gradlew assembleRelease",
     "link": "react-native link",
     "ios": "react-native run-ios",
     "android": "react-native run-android",
+    "android:release": "react-native run-android --variant=release",
+    "android:signkey": "cd android/app/ && keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000 && cd -",
```

Comment:
Yep so I generated the keystore file and it's currently living on my machine. It is being ignored in `.gitignore` and that's a good thing since we shouldn't be committing it in

If I'm not mistaken , `android:signkey` is for _creating_ a new signing key correct? Not sure how or why we should pass in the keystore as a variable here